### PR TITLE
feat(theme): update theme variables and button variants

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: packages/ui-library/src/theme/variants/button-variants.ts [hunk 1]
```diff
@@ -1,112 +1,62 @@
...
-          ...buildButtonDisabledVariants(mode, ownerState),
-          // common attributes
-          textTransform: 'none',
-          padding: `${SPACING(3)} ${SPACING(6)}`,
-        };
+      // root: ({ ownerState }) => {
+      root: {
+        // return {
+        //   ...buildButtonDisabledVariants(mode, ownerState),
+        // common attributes
```
Comment (th0rOdinson) [inline packages/ui-library/src/theme/variants/button-variants.ts:12]: We could get rid of this comments

File: packages/ui-library/src/theme/variants/button-variants.ts [hunk 2]
```diff
@@ -1,112 +1,62 @@
...
+        textTransform: 'none',
+        padding: `${SPACING(3)} ${SPACING(6)}`,
+        // };
       },
       endIcon: {
         marginLeft: `${DEFAULT_SPACING}px`,
       },
-      outlined: {
-        border: `1px solid ${colors[mode].border.border1}`,
+      outlinedPrimary: {
```
Comment (th0rOdinson) [inline packages/ui-library/src/theme/variants/button-variants.ts:20]: Oh this is much better than a secondary button that is outlined 😄 . Nice job!

File: apps/root/src/frame/components/background-grid/light.tsx [hunk 3]
```diff
@@ -0,0 +1,753 @@
+import React from 'react';
+import { CustomSvgIcon, SvgIconProps } from 'ui-library';
+
+interface IconProps extends SvgIconProps {
+  size?: string;
+}
+
+export default function LightBackgroundGrid({ size, ...props }: IconProps) {
+  return (
+    <CustomSvgIcon viewBox="0 0 1921 668" {...props} sx={{ width: '100%', height: '100%' }} fill="none">
```
Comment (th0rOdinson) [inline apps/root/src/frame/components/background-grid/light.tsx:10]: I believe we should split the Grid from the gradient in different approaches. The grid is well positioned, but the gradient should continue bellow the 100vh
Screenshot 2024-03-06 at 14 35 54

I did something similar in the Swap landing!

Comment (FiboApe) [reply apps/root/src/frame/components/background-grid/light.tsx:10]: Oh shit forgot to set it as fixed

Comment (th0rOdinson) [reply apps/root/src/frame/components/background-grid/light.tsx:10]: It's not reaching the bottom yet 😢 
Screenshot 2024-03-06 at 15 27 07

**Review by th0rOdinson — APPROVED** [review]

**Pull request merged**